synfig-core/src/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 ##
 
 # TODO: check if we really need all of them
```

Comment:
Is this comment still needed after this PR?

Reply:
I believe it is safe to remove. @ice0, are you okay with that? ^__^

Reply:
Sure, this can be removed now.